package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "status-vac",
```

Comment:
This should be vac.dev

Reply:
Talked about in Discord

Reply:
[New design](https://vacp2p.netlify.app/presenting-js-waku) for articles is applied